### PR TITLE
Extra Information related to stores and website.

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Extra Information related to stores and website.

### CODE REVIEW

File: guides/v2.1/config-guide/multi-site/ms_over.md [hunk 1]
```diff
@@ -34,13 +36,15 @@ This section discusses the tasks required to use the `MAGE_RUN_TYPE` and `MAGE_R
...
+
+	Using  Website Customer account can be shared with different Website from a single magento instance. One Website contains at least one store. Catalog Prices should be manage at Website level.
 
 *	*Store* is contained by a {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %}. In turn, a store contains at least one *store view*. 
 
-	Multiple stores can share cart, user sessions, payment gateways, and so on, but they have separate catalog structures. 
+	Multiple stores can share cart, user sessions, payment gateways, and so on, but they have separate catalog structures and Catalog price and Catalog Qty can't be manage at store level.
 
-	Store views change the way pages are presented, and are typically used to display a store with different layouts or languages. 
+	Store views change the way pages are presented, and are typically used to display a store with different layouts or languages.You can manage different currency at each store views also.
```
Comment (NamrataChangani) [inline guides/v2.1/config-guide/multi-site/ms_over.md]: @RakeshJesadiya Please add one space after the colon.

File: guides/v2.1/config-guide/multi-site/ms_over.md [hunk 2]
```diff
@@ -34,13 +36,15 @@ This section discusses the tasks required to use the `MAGE_RUN_TYPE` and `MAGE_R
 
 We use the following terms:
 
-*	*Website* is the top-level container for sites, shipping methods, payment methods, and so on. To create completely separate sites that do not share cart, shipping methods, and so on,  you must create separate websites. 
+*	*Website* is the top-level container for sites, shipping methods, payment methods, and so on. To create completely separate sites that do not share cart, shipping methods, and so on,  you must create separate websites.
+
+	Using  Website Customer account can be shared with different Website from a single magento instance. One Website contains at least one store. Catalog Prices should be manage at Website level.
```
Comment (NamrataChangani) [inline guides/v2.1/config-guide/multi-site/ms_over.md]: @RakeshJesadiya Why are you using capitalize in a random place? any reason?

Comment (shrielenee) [reply guides/v2.1/config-guide/multi-site/ms_over.md]: This paragraph is a little confusing. I would propose a rewrite similar to:

"Website customer accounts can be shared between multiple websites within a single Magento instance. A website contains at least one store. Catalog prices should be managed at the website level."

File: guides/v2.1/config-guide/multi-site/ms_over.md [hunk 3]
```diff
@@ -34,13 +36,15 @@ This section discusses the tasks required to use the `MAGE_RUN_TYPE` and `MAGE_R
...
 
 *	*Store* is contained by a {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %}. In turn, a store contains at least one *store view*. 
 
-	Multiple stores can share cart, user sessions, payment gateways, and so on, but they have separate catalog structures. 
+	Multiple stores can share cart, user sessions, payment gateways, and so on, but they have separate catalog structures and Catalog price and Catalog Qty can't be manage at store level.
```
Comment (shrielenee) [inline guides/v2.1/config-guide/multi-site/ms_over.md]: Separate this into two sentences - it is a run-on sentence.

File: guides/v2.1/config-guide/multi-site/ms_over.md (hunk 1 above)
Comment (shrielenee) [reply guides/v2.1/config-guide/multi-site/ms_over.md]: Also reword it to be correct, like "You can manage different currencies per store view."

File: guides/v2.1/config-guide/multi-site/ms_over.md (hunk 2 above)
Comment (shrielenee) [reply guides/v2.1/config-guide/multi-site/ms_over.md]: Good content addition though!

File: guides/v2.1/config-guide/multi-site/ms_over.md (hunk 1 above)
Comment (RakeshJesadiya) [reply guides/v2.1/config-guide/multi-site/ms_over.md]: Done.

File: guides/v2.1/config-guide/multi-site/ms_over.md (hunk 2 above)
Comment (RakeshJesadiya) [reply guides/v2.1/config-guide/multi-site/ms_over.md]: Done. Thanks for appreciating.

**Comment by shrielenee** [comment]
Thanks @RakeshJesadiya! 😃
